docs/en/data_types/enum.md

```diff
@@ -1,13 +1,13 @@
 # Enum
 
-Enumerated type storing pairs of the `'string' = integer` format.
+Enumerated type storing pairs in `'string' = integer` format.
 
 ClickHouse supports:
 
-- 8-bit `Enum`. It can contain up to 256 values with enumeration of `[-128, 127]`.
-- 16-bit `Enum`. It can contain up to 65536 values with enumeration of `[-32768, 32767]`.
+- 8-bit `Enum`. It can contain up to 256 values enumerated in the `[-128, 127]` range.
+- 16-bit `Enum`. It can contain up to 65536 values enumerated in the `[-32768, 32767]` range.
 
-ClickHouse automatically chooses a type for `Enum` at data insertion. Also, you can use `Enum8` or `Enum16` types to be sure in size of storage.
+ClickHouse automatically chooses the type of `Enum` when data is inserted. You can also use `Enum8` or `Enum16` types to lock in the storage size.
```

Comment:
* second sentence does not make sense now
* needs a link to LowCardinality(String) data type as an alternative more flexible approach

Reply:
LowCardinality is not described yet, but I added your comment to the task for the type description.